backend/apps/plugins/tests.py

```
@@ -0,0 +1,71 @@
+            "api_version": "1.0.0",
+            "description": "A simple test plugin.",
+            "author": "Tester",
+            "entry_point": "test_plugin",
+        }
+        manifest = validate_manifest(valid_data)
+        self.assertEqual(manifest.name, "test_plugin")
+
+        # Invalid name validation
+        invalid_data = valid_data.copy()
+        invalid_data["name"] = "Invalid Plugin!"
+        with self.assertRaises(Exception):
```

Ruff (0.15.21)

[warning] Do not assert blind exception: `Exception`

(B017)